### ISSUE TITLE
Facebook add subtitle support

### ISSUE THREAD

**Issue by combat-entropy** [ytdl-org/youtube-dl#22777]
## Checklist

- [x] I'm reporting a site feature request
- [x] I've verified that I'm running youtube-dl version **2019.10.16**
- [x] I've searched the bugtracker for similar site feature requests including closed ones

## Description

Facebook Watch originals like Limetown https://www.facebook.com/Limetown/videos/2106944929612019/ have subtitles (Facebook calls them captions) youtube-dl is currently unable to list/download them. For the linked video --list-subs returns:

```
youtube-dl.py -v --list-subs https://www.facebook.com/Limetown/videos/210
6944929612019/
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', '--list-subs', 'https://www.facebook.com/Limet
own/videos/2106944929612019/']
[debug] Encodings: locale cp1252, fs utf-8, out utf-8, pref cp1252
[debug] youtube-dl version 2019.10.16
[debug] Python version 3.7.4 (CPython) - Windows-7-6.1.7601-SP1
[debug] exe versions: ffmpeg 4.2, ffprobe 4.2
[debug] Proxy map: {}
[facebook] 2106944929612019: Downloading webpage
2106944929612019 has no subtitles
```

**Comment by johnyoonh** [comment]
@remitamine Tried it, but doesn't work:

    youtube-dl -v --list-subs https://www.facebook.com/FacebookforDevelopers/videos/422572928569998/

[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', '--list-subs', 'https://www.facebook.com/FacebookforDevelopers/videos/422572928569998/']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2019.10.22
[debug] Python version 3.6.7 (CPython) - Linux-4.15.0-65-generic-x86_64-with-elementary-5.0-juno
[debug] exe versions: ffmpeg N-93460-g9ae8f3c, ffprobe N-93460-g9ae8f3c, phantomjs 2.1.1, rtmpdump 2.4
[debug] Proxy map: {}
[facebook] 422572928569998: Downloading webpage
422572928569998 has no subtitles

**Comment by remitamine** [comment]
wait the next version:
```
python __main__.py --list-subs https://www.facebook.com/FacebookforDevelopers/videos/422572928569998/
[facebook] 422572928569998: Downloading webpage
Available subtitles for 422572928569998:
Language formats
en       srt
```
